docs/content/en/docs/user-guide/configuration-reference.md

```diff
@@ -198,6 +203,19 @@ One of `yamlField` or `regex` is required.
 | timeout | duration | The maximum time the stage can be taken to run. | No |
 | with | [StageOptions](/docs/user-guide/configuration-reference/#stageoptions) | Specific configuration for the stage. This must be one of these [StageOptions](/docs/user-guide/configuration-reference/#stageoptions). | No |
 
+## DeploymentNotification
+
+| Field | Type | Description | Required |
+|-|-|-|-|
+| mentions | [][NotificationMention](/docs/user-guide/configuration-reference/#notificationmention) | List of users to be notified for each event. | No |
+
+## NotificationMention
+
+| Field | Type | Description | Required |
+|-|-|-|-|
+| event | []string | The event to be notified to users. | No |
+| slack | []string | The user id for mentioning in Slack. | No |
```

Comment:
I think it would be great if we can add some more note here to describe what is `user id` and how to get it. External link from official docs could help in this case, ref [link](https://api.slack.com/enterprise/grid#user_ids) for example

Reply:
0ce72f4